src/index.html

```diff
@@ -6,12 +6,33 @@
 	<title>Chatosaurus</title>
 </head>
 <body>
-
-<h1>My First Heading</h1>
-
-<div id="container"></div>
-
-<img src="./img/nitro-dude.png" alt="img"/>
-
+	<div class="app">
+		<nav class="side">
+			<div class="side-server">
+				<div class="side-server-home"></div>
+				<div class="side-server-channel"></div>
+				<div class="side-server-download"></div>
```

Comment:
What is the intention of these elements? Is there an OOCSS name you could give these items?

Reply:
These are just place holders for now